### DISCUSSION TITLE
refactor: Get React StrictMode Working in Client

### DISCUSSION THREAD

**Discussion by danorlando** [danny-avila/LibreChat#503] [Help Wanted]
The application was originally built without React StrictMode, which allowed for some bad practices within the client application. Currently, wrapping the application with React.StrictMode breaks the application. The fixes should be relatively minor, but the issues that StrictMode is exposing need to be addressed.

**Comment by Cfomodz** [comment]
I believe this discussion is ready to be closed :) 🚀  @danorlando
